src/workerd/io/trace-stream.c++

```diff
@@ -407,6 +404,13 @@ jsg::JsValue ToJs(jsg::Lock& js, const tracing::Onset& onset, StringCache& cache
     }
   }
 
+  // TODO: In prod, always provide this, even if empty (just need to update tests).
```

Comment:
- [ ] In prod, some attributes will always be available, but adding a sample tag to workerd to see that it works there and updating the expected test outputs everywhere is tedious – I will do that before merging this, but it shouldn't block reviews to see if the general approach here is workable.

Reply:
Turns out that the tags I had in mind here aren't easily available in workerd – will scrap this for now, but the work on that did end up going into #4912.